requirements-base.txt

```diff
@@ -49,7 +49,7 @@ redis-py-cluster==1.3.4
 redis>=2.10.3,<2.10.6
 requests-oauthlib==1.2.0
 requests[security]>=2.20.0,<2.21.0
-sentry-relay>=0.5.7,<0.6.0
+sentry-relay>=0.5.9,<0.6.0
```

Comment:
getsentry needs to be updated too, else CI will fail

Reply:
yar. thanks